include/sys/ddt.h

```diff
@@ -69,7 +69,7 @@ typedef struct ddt_key {
 	/*
 	 * Encoded with logical & physical size, and compression, as follows:
 	 *   +-------+-------+-------+-------+-------+-------+-------+-------+
-	 *   |   0   |   0   |   0   | comp  |     PSIZE     |     LSIZE     |
+	 *   |   0   |   0   |   0   |E| comp|     PSIZE     |     LSIZE     |
```

Comment:
I will change the documentation to use X for the bit. I can move it to bit 61 to match blkptr_t, but that seems a bit wasteful since we start breaking up our 3 contiguous 8 bit words. The on-disk format only supports 7 bits of compression types as is so we're not losing anything by doing this.